src/Sylius/Bundle/CoreBundle/Validator/Constraints/ValidCatalogPromotionRulesValidator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
+
+use Sylius\Component\Core\Repository\ProductVariantRepositoryInterface;
+use Sylius\Component\Promotion\Model\CatalogPromotionRuleInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Webmozart\Assert\Assert;
+
+final class ValidCatalogPromotionRulesValidator extends ConstraintValidator
```

Comment:
`Valid...Validator`, it is quite obvious, I would be for simplify it to:
```suggestion
final class CatalogPromotionRulesValidator extends ConstraintValidator
```